hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -1658,6 +1658,13 @@
       DFS_NAMESERVICES_RESOLVER_IMPL =
       "dfs.datanode.nameservices.resolver.impl";
 
+  public static final String
+      DFS_DATANODE_LOCK_MANAGER_TRACE =
+      "dfs.lock.manager.trace";
```

Comment:
Nit: 
a. "DFS_DATANODE_LOCK_MANAGE_TRACE" to "DFS_DATANODE_LOCKMANAGER_TRACE";
b. "dfs.lock.manager.tracer" to "dfs.datanode.lockmanager.trace";
c. "DFS_DATANODE_LOCK_MANAGER_TRACE_DEFAULT" to "DFS_DATANODE_LOCKMANAGER_TRACE_DEFAULT";